docs/source/devguide/xml-parsing.rst

```diff
@@ -1,118 +0,0 @@
-.. _devguide_xml-parsing:
-
-=================
-XML Input Parsing
-=================
-
-OpenMC relies on the FoX_ Fortran XML library for reading and intrepreting the
-XML input files for geometry, materials, settings, tallies, etc. The use of an
-XML format makes writing input files considerably more flexible than would
-otherwise be possible.
-
-With the FoX library, extending the user input files to include new tags is
-fairly straightforward. The steps for modifying/adding input are as follows:
-
-1. Add appropriate calls to procedures from the `xml_interface module`_, such as
-   ``check_for_node``, ``get_node_value``, and ``get_node_array``. All input
-   reading is performed in the `input_xml module`_.
-
-2. Make sure that your input can be categorized as one of the datatypes from
-   `XML Schema Part 2`_ and that parsing of the data appropriately reflects
-   this. For example, for a boolean_ value, true can be represented either by
-   "true" or by "1".
-
-3. Add code to check the variable for any possible errors.
-
-A set of `RELAX NG`_ schemata exists that enables real-time validation of input
```

Comment:
Should this block on RELAXNG be included as a final step in the new "user-input.rst" file?